BUILD/capture_warnings.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+TMP_FILE=/tmp/compile.warning
```

Comment:
Wrong usage of fixed file name in **/tmp**. Use `mktemp`. Use **/dev/shm** to utilize tmpfs. `TMP_FILE` is not needed (see below).